postman/collection-dir/checkout/Flow Testcases/Happy Cases/Scenario14-Save card payment with manual capture/Payments - Capture/event.test.js

```diff
@@ -103,7 +103,7 @@ if (jsonData?.amount_capturable) {
   pm.test(
     "[post]:://payments/:id/capture - Content check if value for 'amount_capturable' matches 'amount - 540'",
     function () {
-      pm.expect(jsonData.amount_capturable).to.eql(6540);
+      pm.expect(jsonData.amount_capturable).to.eql(540);
```

Comment:
[Line 26](https://github.com/juspay/hyperswitch/blob/1cb7e3ea83a06b4802f6b09d9941b5827dc8acfc/postman/collection-dir/checkout/Flow%20Testcases/Happy%20Cases/Scenario14-Save%20card%20payment%20with%20manual%20capture/Payments%20-%20Create/request.json#L26).

Reply:
In capture call, `amount_to_capture` is 6000.
https://github.com/juspay/hyperswitch/blob/1cb7e3ea83a06b4802f6b09d9941b5827dc8acfc/postman/collection-dir/checkout/Flow%20Testcases/Happy%20Cases/Scenario14-Save%20card%20payment%20with%20manual%20capture/Payments%20-%20Capture/request.json#L21

Reply:
Yes, shouldn't here be `6000` instead of `540`?

Reply:
No. This is `amount_capturable` field.
The remaining amount that can be captured (6540 - 6000).